third_party/WebKit/Source/modules/vr/VRAnchorEvent.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#ifndef VRAnchorEvent_h
+#define VRAnchorEvent_h
+
+#include "modules/EventModules.h"
+#include "modules/vr/VRDisplay.h"
+#include "modules/vr/VRAnchorEventInit.h"
+
+namespace blink {
+
+class VRAnchor;
+
+class VRAnchorEvent final : public Event {
+  DEFINE_WRAPPERTYPEINFO();
+
+ public:
+  static VRAnchorEvent* create() { return new VRAnchorEvent; }
+  static VRAnchorEvent* create(const AtomicString& type,
+                              bool canBubble,
+                              bool cancelable,
+                              VRDisplay* display,
+                              HeapVector<Member<VRAnchor>> planes) {
+    return new VRAnchorEvent(type, canBubble, cancelable, display, planes);
+  }
+
+  static VRAnchorEvent* create(const AtomicString& type,
+                              const VRAnchorEventInit& initializer) {
+    return new VRAnchorEvent(type, initializer);
+  }
+
+  ~VRAnchorEvent() override;
+
+  VRDisplay* display() const { return m_display.get(); }
```

Comment:
I am pretty sure you aren't supposed to have spaces within {} blocks like this according to Google style (at least for python, can't find it for c, but check go/cstyle and see if you can find it).